playbooks/appsemblerPlaybooks/eucalyptus_enterprise_deploy.yml

```diff
@@ -112,3 +112,21 @@
     - role: nginx
       nginx_sites:
       - certs
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-0"
+  sudo: True
+  gather_facts: True
+  vars:
+    POSTGRESQL_AGGREGATION_INSTALL_CRON: true
+  roles:
+    - postgresql_aggregation
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-1"
```

Comment:
@tkeemon No, please see Aron comment on the slack channel: https://appsembler.slack.com/archives/C40PW2MU7/p1499391617740405

The cron job should run only in one of the edxapp servers.